packages/airnode-protocol-v1/test/monetization/RequesterAuthorizerWhitelisterWithTokenDeposit.sol.js

```diff
@@ -1241,8 +1302,10 @@ describe('withdrawTokens', function () {
           expect(whitelistStatus.indefiniteWhitelistCount).to.equal(0);
         });
       });
-      context('Withdrawn deposit was not the last one for the requester-endpoint pair', function () {
-        it('decrements the number of times tokens were deposited for the requester-endpoint pair and withdraws tokens', async function () {
+      context('Not signaling intent to withdraw the last deposit for the requester-endpoint pair', function () {
```

Comment:
kind of ambiguous what this means maybe something like:
`signaling intent to withdraw but not for the last deposit for the requester-endpoint pair`
is better

Reply:
Fixed in https://github.com/api3dao/airnode/pull/926/commits/8c97ab69b9784d88131ff5e9444d70db975aa910